docs/DeveloperGuide.adoc

```diff
@@ -347,10 +348,15 @@ _{More to be added}_
 == Non Functional Requirements
 
 .  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
```

Comment:
'Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.' This should be included as its part of the requirement for team project, we can establish it as 'should be able to hold up to 1000 teams/participants/entities(not too sure which one is better, participants sounds good)'. I am not too sure about limiting to 1000 Entities, what do you think? :)

Reply:
It's there